### PR TITLE
84 select size of quiz

### PR DESCRIPTION
**Pull request by BaloeDB**
QuizSizeSelector on frontend
Four options: 5, 10, 20, custom size

Backend create question now with given size

Use create test question for now to test out endpoint

### CODE REVIEW

File: frontend/src/create-quiz/quiz-size-selector/QuizSizeSelector.tsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+            />
+            <InputGroup.Text>{option} Options</InputGroup.Text>
+          </span>
+        ))}
+        <span style={{ ...flex }}>
+          <InputGroup.Radio
+            checked={checked[checked.length - 1]}
+            onChange={() => updateSelectedOptions(checked.length - 1)}
+          />
+          <Form.Control
```
Comment (RaafiKuijper) [inline frontend/src/create-quiz/quiz-size-selector/QuizSizeSelector.tsx:60]: Auto selecting the custom option when changing the value would be nice

**Review by RaafiKuijper — DISMISSED** [review]
Good code overall, but you can create quizzes with 0 questions and quizzes with more question than in the database

**Comment by BaloeDB** [comment]
Add a bit of padding between size selectors.

**Review by NolKeereweer — APPROVED** [review]
Works as intended. The styling looks good. Well done! 
(I like the rename to QuizQuestionCount, shorter and easier to understand)

**Pull request merged**